src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const isMale = person.sex === 'm';
```

Comment:
I think it is better to save the whole condition with ternary operator to some variable like `partner`
```
const partner = person.sex === 'm' ? 'wife' : 'husband';
...
<p className="Person__partner">
  {`${person.partnerName} is my ${partner}`}
</p>